hat/assets/js/apps/Iaso/domains/forms/hooks/useGetPossibleFields.ts

```diff
@@ -78,7 +78,7 @@ export const useDynamicPossibleFields = (
     // Build dynamic queries array for possible fields
     const queries = useMemo(() => {
         return formIds.map(formId => ({
-            queryKey: ['form', formId, 'possible_fields'],
+            queryKey: ['form', `form-${formId}`, 'possible_fields'],
```

Comment:
It would be more consistent to pass a `queryString`.
Also, shouldn't it be `"forms"` (plural)?